hera_cal/redcal.py

```diff
@@ -357,7 +546,7 @@ def compute_ubls(self, data, gain_sols):
             ubl_sols[blgrp[0]] = np.average(d_gp, axis=0)  # XXX add option for median here?
         return ubl_sols
 
-    def logcal(self, data, sol0={}, wgts={}, sparse=False):
+    def logcal(self, data, sol0={}, wgts={}, sparse=False, mode='default'):
```

Comment:
Can you explain `mode` in the docstring?

Reply:
done.